### PR TITLE
Calendar: Fixes unwanted scroll bars on months with exactly 4 weeks

### PR DESCRIPTION
**Pull request by aleksandrjPersonal**
#### Pull request checklist

- [x] Addresses an existing issue: #599 
- [ ] Include a change request file if publishing 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by msftgits** [comment]
Hi, I am closing and re-opening this PR to bump the CLA bot. Sorry for the inconvenience!

**Comment by aleksandrjPersonal** [comment]
@mikewheaton -- Yea, I considered that originally but I was nervous about setting heights on things that didn't already have a height spec'd. I will make the update as you suggested. thank you : )

File: packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss [hunk 1]
```diff
@@ -399,6 +391,7 @@
       @include ms-u-borderBox;
       @include border-right(1px, solid, $ms-color-neutralLight);
       width: 220px;
+      min-height: 230px; // Ensure the same height for months that span 4 and 5 weeks.
```
Comment (mikewheaton) [inline packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss]: Just to really annoy you, I'm not realizing that this is sort of a 'magic number'. Can we make a variable that defines the height of one day, and then use that both for the day and for calculating what six rows is equal to? I wouldn't block on that, but it's something we can do to make sure this doesn't break if the days are made larger in the future. 😄

**Comment by dzearing** [comment]
I think Mike's suggestion is awesome, but this this is jumping around and I'd like to get it fixed sooner than later. Please feel free to submit a separate review with the cleanup.

**Pull request merged**